cmd/identities/get.go

```diff
@@ -29,10 +37,23 @@ func NewGetCmd() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			c := cliclient.NewClient(cmd)
 
+			// we check includeCreds argument is valid
+			for _, opt := range includeCreds {
+				e := stringsx.SwitchExact(opt)
+				if !e.AddCase("oidc") {
+					cmd.PrintErrln(`You have to put a valid value of credentials type to be included, try --help for details.`)
+					return nil
```

Comment:
Returning `nil` here will result in the exit code being 0.
Instead, if you use https://github.com/ory/x/blob/c4721212a9d61b94046379934c1f99bb2f917792/cmdx/helper.go#L31-L37
```suggestion
					return cmdx.FailSilently(cmd)
```
it will not print the error, but fail with non-zero exit code.

Reply:
I wasn't sure about it, now got it, ty!